Kitodo-DataManagement/src/main/resources/db/migration/V2_54__update_LDAP_Group_table.sql

```diff
@@ -29,4 +29,4 @@ sambaPasswordMustChange = '2147483647',
 sambaPasswordHistory = '00000000000000000000000000000000000000',
 sambaLogonHours = 'FFFFFFFFFFFFFFFFFFFF',
 sambaKickoffTime = '0'
-WHERE id = 2 ;
+WHERE id = 2 AND homeDirectory IS NULL AND gidNumber IS NULL AND sambaSID IS NULL;
```

Comment:
does the id needs to be 2?

Reply:
I did not add id with value 2 as this was in the previous and merged pull request. My goal is only not to overwrite existing data.

Reply:
I think default one has name test, so maybe we could use name.

Reply:
I think searching by name is the better idea here.

Reply:
Ok. I will change it.

Reply:
Did not work as flyway migration complain about missing KEY column in WHERE condition.